README.md

```diff
@@ -6,33 +6,33 @@
 
 Fluent UI web represents a collection of utilities, React components, and web components for building web applications.
```

Comment:
![image](https://user-images.githubusercontent.com/14183168/197484939-197d900e-3b31-41af-b48d-286ce0412d76.png)

Is there sense to remove this?

Reply:
I will follow up on this with @paulgildea - for now let's keep it in.